mesa/space.py

```diff
@@ -82,10 +87,10 @@ def __init__(self, width, height, torus):
         self.width = width
         self.torus = torus
 
-        self.grid = []
+        self.grid = []  # type: List[List[GridContent]]
```

Comment:
What do you think of dropping support for Python 3.5 and using the newer type style like
`self.grid: List[List[GridContent]] = []`
Dropping 3.5 would also ease the use of black (well, at least for the build on travis)

Reply:
Python 3.5 is still used in Xenial though, and might break existing code. I don't have strong opinion on whether to drop 3.5 or not.